R/climatic_missing.R

```diff
@@ -0,0 +1,101 @@
+#' climatic_missing
```

Comment:
@EVANSTATS can we have a more informative title here? If you're not sure of a better title, we can always leave it for now and ask Roger at a later date.
From reading the description, I would suggest something like "Summarise missing climatic values"?